### DISCUSSION TITLE
out-of-the-box way to group sidebar nav items

### DISCUSSION THREAD

**Discussion by anwar-al-jahwari** [payloadcms/payload#908] [Feature Requests & Ideas]
A common feature that requested by many payloadcms users is grouping multiple collection into one. 

i would suggest to have one attribute on the collection called  **parent** if it is specified then the collection will go under that tab on the navigation side.

The reasoning for the approach is to make the least change as possible to the base code.

**Comment by TheTimeWalker** [comment]
I think the feature you're asking should be implemented in v1.1.0 now. You only have to define the `group` variable in the admin object like this:

```
  admin: {
    useAsTitle: "email",
    group: "Admin",
  }
```
